hack/e2e-test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -e
+
+component=hive
+TEST_IMAGE=$(eval "echo $IMAGE_FORMAT")
+component=installer
+INSTALLER_IMAGE=$(eval "echo $IMAGE_FORMAT")
+
+
+# Install Hive
+make deploy DEPLOY_IMAGE="${TEST_IMAGE}"
+
+CLUSTER_PROFILE_DIR=/tmp/cluster
+
+# Create a new cluster deployment
+export BASE_DOMAIN="origin-ci-int-aws.dev.rhcloud.com"
+export CLUSTER_NAME="$(oc get cluster.cluster.k8s.io -n openshift-cluster-api -o jsonpath='{ .items[].metadata.name }')-1"
+export SSH_PUB_KEY="$(cat ${CLUSTER_PROFILE_DIR}/ssh-publickey)"
+export PULL_SECRET="$(cat ${CLUSTER_PROFILE_DIR}/pull-secret)"
+export AWS_ACCESS_KEY_ID=$(cat ${AWS_SHARED_CREDENTIALS_FILE} | awk '/aws_access_key_id/ { print $3; exit; }')
+export AWS_SECRET_ACCESS_KEY=$(cat ${AWS_SHARED_CREDENTIALS_FILE} | awk '/aws_secret_access_key/ { print $3; exit; }')
+
+function teardown() {
+	echo "Deleting ClusterDeployment ${CLUSTER_NAME}"
+	oc delete clusterdeployment ${CLUSTER_NAME}
```

Comment:
You probably need a waitforjob here as well.

Reply:
This is tricky because deprovision will immediately kill the install job, which means CI probably won't have our pod logs saved when it finishes up. Should we add or re-use an annotation to keep the install job around, disassociated with the cluster delpoyment?